src/content/get-started/overview.mdx

```diff
@@ -51,12 +57,46 @@ Be sure to configure your kubectl to talk to your newly minted cluster.
 
 Follow the official documentation for [installing the latest release of Helm v3 (v3.8 or higher)](https://helm.sh/docs/intro/install/).
 
-### License
+## Deploying Okteto
 
-You'll receive a license key as part of your subscription to Okteto. If you haven't received it, [please open a support ticket](https://okteto.com/support).
+Okteto is distributed as a Helm chart. This contains all the required components for the complete Okteto experience.
 
-If you are interested in evaluating Okteto, [sign up for our free 30 days trial.  No credit card required](https://www.okteto.com/free-trial/).
+Okteto components:
 
-## Deploying Okteto
+1. NGINX Ingress
+1. Okteto/API
+1. Okteto/Buildkit
+1. Okteto/Frontend
+1. Okteto/Migration
+1. Okteto/Registry
+1. Okteto/Validation Webhook
```

Comment:
I don't think that we need to speficy the okteto-level components at this level. This is the first time the prospect sees the helm installation, this level of detail is not necessary. I'd prefer to have a separate page where we explain the components, architecture, etc.